doc/wallet-backup.md

```diff
@@ -2,90 +2,142 @@
 
-For all methods, you will need to include an export directory setting in your config file (`zcash.conf` located in the data directory which is `~/.zcash/` unless it's been overridden with `datadir=` setting):
+For all methods, you will need to include an export directory setting in your
+config file (`zcash.conf` located in the data directory which is `~/.zcash/`
+unless it's been overridden with `datadir=` setting):
 
 `exportdir=path/to/chosen/export/directory`
 
-You may chose any directory within the home directory as the location for export & backup files. If the directory doesn't exist, it will be created.
+You may chose any directory within the home directory as the location for
+export & backup files. If the directory doesn't exist, it will be created.
 
-Note that zcashd will need to be stopped and restarted for edits in the config file to take effect. 
+Note that zcashd will need to be stopped and restarted for edits in the config
+file to take effect.
 
 ### Using `backupwallet`
 
 To create a backup of your wallet, use:
 
-`zcash-cli backupwallet <nameofbackup>`.
+```bash
+$ zcash-cli backupwallet <nameofbackup>
+```
 
-The backup will be an exact copy of the current state of your wallet.dat file stored in the export directory you specified in the config file. The file path will also be returned.
+The backup will be an exact copy of the current state of your wallet.dat file
+stored in the export directory you specified in the config file. The file path
+will also be returned.
 
-If you generate a new Zcash address, it will not be reflected in the backup file.
+If you generate a new Zcash address, it will not be reflected in the backup
+file.
 
-If your original `wallet.dat` file becomes inaccessible for whatever reason, you can use your backup by copying it into your data directory and renaming the copy to `wallet.dat`.
+If your original `wallet.dat` file becomes inaccessible for whatever reason,
+you can use your backup by copying it into your data directory and renaming the
+copy to `wallet.dat`.
 
 ### Using `z_exportwallet` & `z_importwallet`
 
-If you prefer to have an export of your private keys in human readable format, you can use:
+If you prefer to have an export of your private keys in human readable format,
+you can use:
 
-`zcash-cli z_exportwallet <nameofbackup>`
+```bash
+$ zcash-cli z_exportwallet <nameofbackup>`
+```
 
-This will generate a file in the export directory listing all transparent and shielded private keys with their associated public addresses. The file path will be returned in the command line.
+This will generate a file in the export directory listing all transparent and
+shielded private keys with their associated public addresses. The file path
+will be returned in the command line.
 
 To import keys into a wallet which were previously exported to a file, use:
 
-`zcash-cli z_importwallet <path/to/exportdir/nameofbackup>`
+```bash
+$ zcash-cli z_importwallet <path/to/exportdir/nameofbackup>
+```
 
```

Comment:
This need work. HD-generated keys won't be included in the export file and therefore won't be imported; this includes all Sapling and Orchard keys generated since and including zcashd 4.7.0.